docs/guides/getting-started-orchestrate-apis.md

```diff
@@ -10,13 +10,22 @@ keywords:
 <span class="badge badge--beginner">Beginner</span>
 <span class="badge badge--medium">Time estimate: 15 minutes</span>
 
-A Connector is a reusable building block that works out of the box. Each Connector task can be configured with domain-specific parameters without implementing custom business logic.
+This guide will walk you through working with a REST Connector task as a first time Camunda 8 user. The REST Connector is a [protocol Connector](/docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md#protocol-connectors), where you can make a request to a REST API and use the response in the next steps of your process.
 
-The concept of a Connector consists of two parts: the business logic is implemented as a job worker, and the user interface during modeling is provided using an element template.
+:::note
+New to Connectors? Review our [introduction to Connectors](/docs/components/connectors/introduction.md) to get familiar with their capabilities, and have a closer look at all of the available [out-of-the-box Connectors](/docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md).
+:::
 
-This guide will walk you through working with a REST Connector task as a first time Camunda 8 user.
+A **Connector** is a reusable building block that performs integration with an external system to orchestrate and complete work. For example, you can notify of pending or completed processes through communication tools such as [Slack](/docs/components/connectors/out-of-the-box-connectors/slack.md) or [Microsoft Teams](/docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md), and create or update records in enterprise business systems.
```

Comment:
I would remove this set of sentences. I think its possibly confusing and we route folks to the intro in the admonition above.

Reply:
Resolved in latest commit 👍